src/Exception/OgRoleRequiredException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\og\Exception;
+
+/**
+ * Exception thrown when a required role is compromised.
```

Comment:
It has become "Exception thrown when an invalid action is performed on immutable properties.", is that OK?